app/main.py

```diff
@@ -1,3 +1,71 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float = 0.0) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float)) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + float(other))
+        return NotImplemented
+
+    def __iadd__(self, other: (int, float)) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += float(other)
+        else:
+            return NotImplemented
+        return self
+
+    def __mul__(self, other: (int, float)) -> "Distance":
+        if isinstance(other, (int, float)):
+            return Distance(self.km * float(other))
+        return NotImplemented
+
+    def __truediv__(self, other: (int, float)) -> "Distance":
+        if isinstance(other, (int, float)) and other != 0:
```

Comment:
Try to divide this check to two separate, we have some special case for raising errors when we have dividing to zero